client/src/components/recipe-modal/RecipeModal.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+import { v1 as uuidv1 } from 'uuid';
+
+function RecipeModal(props) {
+  const { recipe } = props;
+
+  const { togglePopUp } = props;
```

Comment:
can combine line 6 and line 8 I think

Reply:
adjsutments made

Reply:
oh I meant combining the object destructuring

`const { recipe, togglePopUp } = props;`

Reply:
ok adjustments made